### DISCUSSION TITLE
Merging pull requests

### DISCUSSION THREAD

**Discussion by channaROXX** [community/community#131903] [Mobile] [answered]
### Select Topic Area

Question

### Body

How do i handle merging pull requests from the GitHub Mobile app?

**Comment by SolutionsCrafter** [comment]
Open the pull request in the app, review the changes, and check the status of checks and reviews. If everything looks good, you can merge the pull request using the merge button. Choose the appropriate merge method (merge commit, squash and merge, or rebase and merge) based on your project’s workflow.

**Comment by chamishkadilina** [comment, marked as answer]
To merge a pull request using the GitHub Mobile app:

+ Open the GitHub Mobile App.
+ Navigate to the Repository containing the PR.
+ Go to the "Pull Requests" tab.
+ Select the Pull Request you want to merge.
+ Review the Pull Request details and checks.
+ Scroll down and tap "Merge pull request".
+ Choose the merge method (Merge, Squash, or Rebase) and confirm the merge.
+ Optionally, delete the branch after merging
